client/src/API/AuthMethods.ts

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+import { LoginData, UserInfoData, URL } from "./Constants";
+
+export const signUp = async (userInfo: UserInfoData) => {
```

Comment:
@Solshuk and @bretpeters3n, this is our signUp method.  It takes an argument of userInfo which is of type UserInfoData.  Please make sure that after the data is validated it is put into this shape and passed into signUp.

Similar process for the logIn method.